.github/workflows/apptek_hashes.yml

```diff
@@ -0,0 +1,49 @@
+name: AppTek hashes
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  test-hashes:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+    - name: Start Bitbucket Pipeline
+      run: |
+        curl -X POST -s -u ${{ secrets.MICHEL_BITBUCKET_AUTHENTICATION }} \
+            -H 'Content-Type: application/json' \
+            https://api.bitbucket.org/2.0/repositories/omnifluent/apptek_asr/pipelines/ \
+            -d '{"target": {"ref_type": "branch", "ref_name": "willi-i6_core-integration",
+                 "type": "pipeline_ref_target", "selector": {"type": "custom", "pattern": "hashes-only"}},
+                 "variables": [{"key": "i6_core_branch", "value": "'${GITHUB_HEAD_REF}'"}]}' |\
+        python -c "import sys,json; print(json.load(sys.stdin)['uuid'])" |\
```

Comment:
You can do json processing the linux way with https://stedolan.github.io/jq/ , no need for python. Also: if you are loading python you could do the postprocessing there and not in sed.

Reply:
I wasn't sure if jq was available.. but then I could also just install it in the pipeline :D

Reply:
please do

Reply:
done